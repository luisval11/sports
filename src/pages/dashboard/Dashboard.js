import React from "react";
import { Banner } from "../../atoms/banner/Banner";
import { Footer } from "../../atoms/footer/Footer";
import { TableScores } from "../../templates/table/TableScores";

const Dashboard = () => {
	return (
		<>
			<Banner />
			<TableScores />
			<Footer />
		</>
	);
};

export default Dashboard;
