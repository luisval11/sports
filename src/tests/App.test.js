import { shallow } from "enzyme";
import Dashboard from "../pages/dashboard/Dashboard";

describe("Test on Dashboard", () => {
	let wrapper = shallow(<Dashboard />);

	test("should match initial Snapshot", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
