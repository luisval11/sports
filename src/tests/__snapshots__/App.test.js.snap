// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Dashboard should match initial Snapshot 1`] = `
<Fragment>
  <Banner />
  <TableScores />
  <Footer />
</Fragment>
`;
